specs/change_password/01_main.rst

```diff
@@ -0,0 +1,174 @@
++--------+-----------------+
+| story  | title           |
++--------+-----------------+
+| 74_    | Change password |
++--------+-----------------+
+
+.. _74: https://github.com/gnosis/safe/issues/74
+
+Change password
+===============
+
+1. `Main`_
+2. `User Interface`_
+3. `External Communication`_
+4. Other_
+
+.. _Main:
+
+.. contents:: Table of Contents
+
+1. Problem Definition
+---------------------
+
+* User wants to change the current app password.
+* Current password was compromised and needs to be changed.
+
+2. Inputs
+-----------
+
+- Current password.
+    - Minimum 8 characters (>=8)
+    - At least 1 number
+    - At least 1 letter
+    - No more than 2 identical characters in a row
+      (e.g. 111 is not allowed, 11 is fine)
+    - Maximum lenght?
```

Comment:
For security reasons, I'd remove all the requirements for the current password - we just need to check if the password matches the one we already have registered. Instead, I would move them to the "New password" item

Reply:
Ya, not sure why I put it here. 🤦‍♂️